patchwork/api/patch.py

```diff
@@ -425,7 +425,17 @@ def partial_update(self, request, *args, **kwargs):
         req_user_id = request.user.id
         is_maintainer = request.user.is_authenticated and (
             obj.project in request.user.profile.maintainer_projects.all()
+            or request.user.is_superuser
         )
+        non_attention_set_keys = [
+            key for key in request.data if key != 'attention_set'
+        ]
+
+        if non_attention_set_keys:
+            if not is_maintainer:
+                raise PermissionDenied(
+                    detail='You do not have permission to edit patch properties.'
+                )
```

Comment:
Wouldn't this be equivalent?
```suggestion
        if non_attention_set_keys and not is_maintainer:
            raise PermissionDenied(
                detail='You do not have permission to edit patch properties.'
            )
```

Reply:
Nice! I'll improve it.